doc/ref_model/chapters/chapter04.md

```diff
@@ -282,43 +282,75 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
 
 Compute Flavours represent the compute, memory, storage, and management network resource templates that are used by VMs on the compute hosts. Each VM is given a compute Flavour (resource template), which determines the VMs compute, memory and storage characteristics.
 
+Customised (Parameterized) Flavors can be used in concession by operators and, if needed, and can be  created using TOSCA, HEAT templates, and/or VIM APIs. These compute Flavours are intended to be used for transitional purposes and workload vendors are expected to consume smaller Flavours and adopt microservices-based designs for their workloads.<br>
```

Comment:
So as to avoid confusion with Node Flavours ([RM Chapter 02](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter02.md#242-node-flavours-specialisations)) we should use "Compute Flavours" instead of just "Flavours". 
Should we have a definition section of terms used early on in this Chapter?
```suggestion
Parameterized Compute Flavours can be used by operators and specified in request for resources. 
```